pkg/cmd/server/admin/create_client.go

```diff
@@ -69,7 +71,7 @@ func NewCommandCreateClient(commandName string, fullName string, out io.Writer)
 
 	flags.StringVar(&options.APIServerURL, "master", "https://localhost:8443", "The API server's URL.")
 	flags.StringVar(&options.PublicAPIServerURL, "public-master", "", "The API public facing server's URL (if applicable).")
-	flags.StringVar(&options.APIServerCAFile, "certificate-authority", "openshift.local.config/master/ca.crt", "Path to the API server's CA file.")
+	flags.StringVar(&options.APIServerCAFile, "certificate-authority", path.Join(defaults.NewDefaults().MasterConfigDir(), "ca.crt"), "Path to the API server's CA file.")
```

Comment:
Too complicated - again this is something that I would argue we would just create in the current directory, or take a directory option.